test/cpp/jit/test_gpu.cpp

```diff
@@ -4477,6 +4477,39 @@ void testGPU_FusionUnrollWithAlloc() {
       aten_output.sub(outputs[0]).abs().max());
 }
 
+void testGPU_FusionIsOneInt() {
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  // Test isOneInt
+  {
+    Int* x = new Int(1);
+    Int* y = new Int(1);
+    Val* z = mul(x, y);
+    TORCH_CHECK(x->isOneInt());
+    TORCH_CHECK(y->isOneInt());
+    if (z->isOneInt()) {
+      std::cout << "z is one int: " << z << std::endl;
+    } else {
+      std::cout << "z is not one int: " << z << std::endl;
+    }
+  }
+
+  // Test isZeroInt
```

Comment:
please move this one to a dedicated test case

Reply:
Done.